org.eclipse.scanning.test/src/org/eclipse/scanning/test/command/ScanRequestCreationTest.java

```diff
@@ -87,7 +88,7 @@ public void before() throws Exception {
 	}
 
 	@Test
-	public void testGridCommandWithROI() throws Exception {
+	public void testGridCommandWithROI() throws GeneratorException {
 		pi.exec("sr =                             "
 			+	"scan_request(                    "
 			+	"    grid(                        "
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "    grid(                        " 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)